spec/models/video_spec.rb

```diff
@@ -39,4 +40,18 @@
 
   end
 
+  describe "#average_rating" do
+    it "should return No reviews yet when there are no reviews" do
```

Comment:
I recommend to leave "should" word from all your specs and examples. It does not say anything useful - rather use 3rd person singular for the verb following 'it' - read more here - http://betterspecs.org/#should